pages/national_resources/gr_resources.md

```diff
@@ -1,5 +1,15 @@
 ---
 
+training:
+  - name: ELIXIR GREECE Zenodo
```

Comment:
for name, it should be title of publication (on Zenodo) and description should be a short summary (2-3 lines on what it is about)